platform/ios/PolyPodApp/PolyPod/ErrorUploader.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import MessagePack
+
+class ErrorUploader {
+    static let shared = ErrorUploader()
+
+    func uploadToServer(_ errorMsg: String,
+                        completionHandler: @escaping (String?, String?) -> Void
+    ) {
+        let endpointId = "polyApiErrorReport"
+
+        guard let endpointInfo = PodApi.shared.endpoint.endpointInfoFromId(endpointId: endpointId) else {
+            Log.error("uploadToServer failed: No endpoint found for: \(endpointId)")
+            completionHandler(
+                "uploadToServer",
+                "\(#function): No endpoint found for \(endpointId)"
+            )
+            return
+        }
+        let payload = jsonStringify([errorMsg])
```

Comment:
Why is it that in iOS code the `errorMsg` is wrapped in an array, while in Android code it isn't?